src/Reporters/ConsoleReporter.ts

```diff
@@ -30,10 +30,13 @@ export default class ConsoleReporter {
 
   public onTestEnd(test: Mocha.Test): void {
     const titlePath = test.titlePath();
+    const titlePathWithoutExtra = titlePath.filter(
+      (title) => title !== 'extra' && title !== 'Extra',
+    );
 
     this.getTestRun().recordTest(
-      titlePath.filter((title) => title !== 'extra' && title !== 'Extra').join(' > '),
-      test.title,
+      titlePathWithoutExtra.join(' > '),
+      titlePathWithoutExtra.slice(1).join(' > '),
```

Comment:
I'm not super happy about this solution. If you have a better idea, please let me know or take over :)

Reply:
I am actually not sure if we should solve the problem like this. I see now why it occurred, I initially was thinking that the tests displayed in the list should be as short as possible because when you add all nested describes, it's just repeated all the time and harder to read.

So instead of having one big group with all the tests, I would recommend having separate smaller groups. That's maybe also easier for competitors to see their points or in which part they still have many tests missing.

Example with one big group:
![image](https://user-images.githubusercontent.com/2709252/132393772-d5278936-9674-4f5c-bb0f-51a1680c38c9.png)


Same split into smaller groups:
![image](https://user-images.githubusercontent.com/2709252/132393675-6eb4ea7b-b2d0-4ef4-962a-4f974fe16d69.png)


Nevertheless, I will merge this as it is since it will solve the error we are facing and I don't have too much time to think about a different way today.

For the future, we may want to think about how we map nested test names to groups, maybe there is a better solution in general.